imap_processing/lo/l0/science_direct_events.py

```diff
@@ -0,0 +1,202 @@
+"""L1A Science Direct Events data class."""
+from dataclasses import dataclass
+from itertools import compress
+
+import bitstring
+from bitarray import bitarray
+
+import imap_processing.lo.l0.compression_tables as ct
+from imap_processing.ccsds.ccsds_data import CcsdsData
+from imap_processing.lo.l0.lol0 import LoL0
```

Comment:
This is hard to parse with all the module levels (also the 0s and Os), and also makes me want to laugh out loud 😂 

I have a personal preference for the single level heirarchy since we don't have that many files in each level, but realize we are split right now in how we are doing that for all the instruments.

```bash
imap_processing.lo.compression_tables
imap_processing.lo.level0
```

Reply:
```python
from imap_processing.lo.l0.lol0
```
In my head I've been calling this my Flo Rida import because whenever I see it I just hear the chorus to Low.

You're right though, I should make it less ridiculous.

Reply:
If you want to keep the `l0` folder, you could rename it `LoBase` or something. Or maybe we can convince everyone to rename the instrument, because it's just going to be rough with L0 and Lo 😆

Reply:
Yeah, every time I see someone's PR or issue that has L0 I think "why are they working on Lo?" and then I realize it's a "0" and not a "o"